src/connectors/stats/connectStats.js

```diff
@@ -108,6 +110,10 @@ export default function connectStats(renderFn, unmountFn = noop) {
       return {
         hitsPerPage: results.hitsPerPage,
         nbHits: results.nbHits,
+        nbSortedHits: results.nbSortedHits,
+        isSmartSorted:
+          typeof results.appliedRelevancyStrictness !== 'undefined' &&
```

Comment:
if it's undefined it's not bigger than 0, so this first condition actually isn't needed I think

Reply:
makes sense https://github.com/algolia/instantsearch.js/pull/4649/commits/a3b67610d49f025e8e032a2c7961cd32cb0883b1

Reply:
I just remembered why I did it so. Even though it's in JS, but when in TS, it gives an error saying it can be undefined so I had to guard it too. So I just reverted the change to get back the undefined check